pgd_splicer/ProcessPDBTask.py

```diff
@@ -215,27 +219,28 @@ def process_pdb(data):
     # added to it as the protein is processed.  This prevents memory leaks
     # due to the original dict having a reference held outside this method.
     # e.g. if it were looped over with a large list of PDBs
-    data = data.copy()
+    # data = data.copy()
+    data = pdb['data']
+    logger = pdb['logger']
 
     try:
         residue_props = None
         code = data['code']
         chains_filter = data.get("chains")
-        filename = 'pdb%s.ent.gz' % code.lower()
-        print '    Processing: ', code
+        logger.debug('process_pdb')
 
         # update datastructure
         data['chains'] = {}
 
         # 1) parse with bioPython
-        data = parseWithBioPython(filename, data, chains_filter)
+        data = parseWithBioPython(code, data, chains_filter)
 
         # 2) Create/Get Protein and save values
         try:
             protein = ProteinModel.objects.get(code=code)
-            print '  Existing protein: ', code
+            logger.debug('protein exists')
```

Comment:
Should there be a bit more information here about *which* protein exists or does not exist?